README.md

```diff
@@ -1,37 +1,67 @@
 # RepSeP
 
-The **Rep**roducible **Se**lf **P**ublishing toolkit demonstrates how to reuse your favourite data analysis workflows in order to seamlessly include quasi-dynamic publication-quality output (e.g. figures, tables, or statistic reports) in the most common science communication formats. Currently we provide examples for:
+The **Rep**roducible **Se**lf **P**ublishing toolkit demonstrates how to reuse
+your favourite data analysis workflows in order to seamlessly include
+quasi-dynamic publication-quality output (e.g. figures, tables, or statistic
```

Comment:
Could you please unwrap the lines? The general style we use is one sentence, one line, just like with code, and like we do in the TeX files.